lib/haml_lint/reporter.rb

```diff
@@ -5,12 +5,15 @@ module HamlLint
   # @abstract
   class Reporter
     attr_reader :lints
+    attr_reader :files
 
     # @param logger [HamlLint::Logger]
     # @param report [HamlLint::Report]
-    def initialize(logger, report)
+    # @param files  [Array]
+    def initialize(logger, report, files)
       @log = logger
       @lints = report.lints
+      @files = files
```

Comment:
This never gets used from what I can see (you can always extract the list of relevant files from the list of `lints`). It can safely be removed.